src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -694,8 +694,15 @@ public void createSctpConnection()
         // Create the SctpManager and provide it a method for sending SCTP data
         this.sctpManager = new SctpManager(
                 (data, offset, length) -> {
+                    // The buffer we get here came from usrsctp, and therefore
+                    // shouldn't be returned to the pool.  We can either
+                    // never return any packets from this path, or copy here
+                    // (actually, ideally, closer to SCTP, since this is
+                    // usrsctp specific) into a pooled buffer.
+                    byte[] newBuf = ByteBufferPool.getBuffer(length);
+                    System.arraycopy(data, offset, newBuf, 0, length);
                     PacketInfo packet
-                        = new PacketInfo(new UnparsedPacket(data, offset, length));
+                        = new PacketInfo(new UnparsedPacket(newBuf, 0, length));
```

Comment:
Packets coming from SCTP are in buffers allocated by usrsctp (as far as I can tell), so we either have to have a way to know not to return them, or, we put them in a pooled buffer.  Here I'm doing the latter, which seemed like a good place to start: I hope the traffic won't be too significant, but if we find it's a problem we can improve it later.  Before merging, though, I would like to put the work of copying into a pooled buffer "closer" to the SCTP code, if possible.